samples/OldReference/InstrumentedHttpCall.cs

```diff
@@ -24,7 +24,9 @@ namespace OldReference;
 
 public static class InstrumentedHttpCall
 {
-    public static async Task GetAsync(string url)
+    private static readonly string OperationName = $"{nameof(InstrumentedHttpCall)}.{nameof(GetAsync)}";
+
+    static InstrumentedHttpCall()
     {
         Console.WriteLine(">>>>>>>>>>>>>>>>>>>>>>> System.Diagnostics.DiagnosticSource assemblies loaded:");
```

Comment:
Do we have an issue to remove these Console.WriteLine calls? I initially thought that we might remove them as part of the POC cleanup.

Reply:
Nevermind, I just realized that this was part of the sample app, so it makes sense to keep this code.